benches/me.rs

```diff
@@ -33,13 +33,13 @@ fn new_plane<T: Pixel>(ra: &mut ChaChaRng, width: usize, height: usize) -> Plane
   p
 }
 
-fn bench_get_sad(b: &mut Bencher, bs: &BlockSize) {
+fn bench_get_sad(b: &mut Bencher, input_type: &(BlockSize, usize)) {
+  let (bs, bit_depth) = input_type;
```

Comment:
I'd prefer these as two parameters to the function, rather than as one `input_type` tuple.

Reply:
I think we are indeed limited by the `bench_function_over_inputs` function that takes a single `inputs` array as parameter.